exploration/percent-format.md

```diff
@@ -237,6 +331,14 @@ Examples using `:unit`, each of which would format as "Completion: 50%.":
 {{Completion: {$n :unit unit=percent}.}}
 ```
 
+_Pros_
+- Avoids multiplication of random values
+- Useful for other scaling operations
+
+_Cons_
+- Might require changes to digit size options, since negative exponents are a Thing
```

Comment:
Rather, this would not be a _digit size option_.
```suggestion
- Cannot use _digit size option_ as the `exp` option value type, since negative exponents are a Thing
```